.env

```diff
@@ -88,6 +88,7 @@ GEOSERVER_INITIAL_MEMORY=512M
 GEOSERVER_MAXIMUM_MEMORY=4G
 # In prod, should switch to PRODUCTION_LOGGING. For other logging profiles see: https://docs.geoserver.org/stable/en/user/configuration/globalsettings.html#config-globalsettings-log-profile
 GEOSERVER_LOGGING_PROFILE=DEFAULT_LOGGING
+GEOSERVER_PLUGIN_DYNAMIC_URLS=https://www.systect.de/downloads/geoserver-2.25.2-vectortiles-plugin.zip
```

Comment:
How do we make sure this external dependency is kept up-to-date? Tools like Dependabot & Renovate Bot won't notify us.

- Is there no other (trustworthy) canonical download server that offers the plugin?
- We could add a task "check for latest version, update `systect.de`" to the list of periodic maintenance tasks, and document this in `ipl-admin-guide`?
- Does Geoserver support reading `file:` URLs? Then we could check the plugin into Git and mount it into the container. It wouldn't help us with getting notified about updates, but it would at least make the setup more self-contained.

Reply:
> - Is there no other (trustworthy) canonical download server that offers the plugin?
Not that I know of.
> - We could add a task "check for latest version, update systect.de" to the list of periodic maintenance tasks, and document this in ipl-admin-guide?
We have the LOP-list for periodic tasks, let's add it there. In addition, we can add a reminder comment besides the `GEOSERVER_IMAGE` tag. Updating this usually would result in an updated version. 
> - Does Geoserver support reading file: URLs? Then we could check the plugin into Git and mount it into the container. It wouldn't help us with getting notified about updates, but it would at least make the setup more self-contained.
I'd like to avoid this for now.

Reply:
Another point to keep in mind: Does Geoserver start if `www.systect.de` is unavailable?